tests/foreman/conftest.py

```diff
@@ -119,63 +131,90 @@ def log_test_execution(robottelo_logger, request):
     robottelo_logger.debug('Finished Test: {}'.format(test_full_name))
 
 
-class NestedDict(SimpleNamespace):
-    def __init__(self, dict_data, **kwargs):
-        super().__init__(**kwargs)
-        for key, value in dict_data.items():
-            if isinstance(value, dict):
-                self.__setattr__(key, NestedDict(value))
-            else:
-                self.__setattr__(key, value)
-
-
-def pytest_configure():
-    """return dict of name->object to be made globally available in
-    the pytest configure.  This hook is called at plugin registration
-    time.
-    Object is accessible only via dotted notation `item.key.nested_key`
-
-    Exposes the list of all WONTFIX bugs and a mapping between decorated
-    functions and Bug IDS (populated by decorator).
-    """
-    log("Registering custom pytest_configure")
-    pytest.bugzilla = NestedDict({
-            'removal_ids': get_deselect_bug_ids(log=log),
-            'decorated_functions': []
-    })
+def generate_bz_collection(items, config, filename):
+    """Generates a dictionary with the usage of BZ blockers"""
 
+    bzcollect_data = {'skip': [], 'deselect': [], 'workaround': []}
 
-def _extract_setup_class_ids(item):
-    setup_class_method = getattr(item.parent.obj, 'setUpClass', None)
-    return getattr(setup_class_method, 'bugzilla_ids', [])
+    for item in items:
+        filepath, lineno, testcase = item.location
+        # Take the module level :CaseComponent:
+        module_component = None
+        with open(os.path.join(os.path.sep, filepath), 'r') as modulefile:
```

Comment:
Was that here before? because now when I try to run it, I get exception:
```
$ pytest tests/foreman/api/ --bzcollect
...
INTERNALERROR>   File "/home/mzalewsk/.virtualenvs/robottelo/lib64/python3.6/site-packages/pluggy/callers.py", line 187, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/home/mzalewsk/sources/robottelo/tests/foreman/conftest.py", line 201, in pytest_collection_modifyitems
INTERNALERROR>     generate_bz_collection(items, config, filename=bzcollect)
INTERNALERROR>   File "/home/mzalewsk/sources/robottelo/tests/foreman/conftest.py", line 143, in generate_bz_collection
INTERNALERROR>     with open(os.path.join(os.path.sep, filepath), 'r') as modulefile:
INTERNALERROR> FileNotFoundError: [Errno 2] No such file or directory: '/tests/foreman/api/test_activationkey.py'
```

Reply:
@mirzal I just pushed a change to use the `module.__file__` as looks like the `item.location` is not absolute.